fiddler/pyproject.toml

```diff
@@ -30,15 +28,14 @@ classifiers = [
 ]
 dependencies = [
     "datadog-checks-base>=25.1.0",
+    "fiddler-client",
 ]
 dynamic = [
     "version",
 ]
 
 [project.optional-dependencies]
```

Comment:
Based on looking at other integrations, I think we need to keep this and remove `fiddler-client` from the `dependencies` field.

Reply:
Agreed